simple/memcached.go

```diff
@@ -0,0 +1,48 @@
+package simple
```

Comment:
I am on the fence about this being in the `simple` package, but it didn't feel like it belonged in the `operator` package either.